SFS_general/virtual_secondary_source_positions.m

```diff
@@ -69,7 +69,7 @@
 narginchk(nargmin,nargmax);
 isargxs(xs);
 if ~isempty(x0)
-  isargsecondarysource(x0);
+    isargsecondarysource(x0);
 end
```

Comment:
The function works if `x0 = []` is provided, but I was wondering if this is meaningful. It basically skips `consider_secondary_sources` which could also be achieved via the `conf` setting. What would be a scenario where you would call this function with an empty `x0`?